neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -122,25 +184,101 @@ def create_experiment(self,
             else:
                 raise
 
+        experiment = self._convert_to_experiment(api_experiment, project)
+        # Initialize new experiment
+        self._execute_alpha_operation(
+            experiment=experiment,
+            operations=self._get_init_experiment_operations(entrypoint, params, tags),
+        )
+        return experiment
+
+    @with_api_exceptions_handler
+    def send_channels_values(self, experiment, channels_with_values):
+        send_operations = []
+        for channel_with_values in channels_with_values:
+            # TODO: handle other data types
+            # points = [Point(
+            #     timestampMillis=int(value.ts * 1000.0),
+            #     x=value.x,
+            #     y=Y(numericValue=value.y.get('numeric_value'),
+            #         textValue=value.y.get('text_value'),
+            #         inputImageValue=value.y.get('image_value'))
+            # ) for value in channel_with_values.channel_values]
+            ch_values = [
+                alpha_operation.LogStrings.ValueType(
+                    value=value.y.get('text_value'),
+                    step=None,
+                    ts=int(value.ts),
+                )
+                for value in channel_with_values.channel_values
+            ]
+            send_operations.append(alpha_operation.LogStrings(
+                path=alpha_path_utils.parse_path(channel_with_values.channel_id),
+                values=ch_values,
+            ))
+
+        self._execute_alpha_operation(experiment, send_operations)
+
+    @with_api_exceptions_handler
     def get_system_channels(self, experiment):
-        return list()
+        params = {
+            'experimentId': experiment.internal_id,
+        }
+        try:
+            experiment = self.leaderboard_swagger_client.api.getExperimentAttributes(**params).response().result
+        except HTTPNotFound:
+            # pylint: disable=protected-access
+            raise ExperimentNotFound(
+                experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id)
+        return [
+            AlphaChannelWithLastValue(
+                ch_id=attr.stringSeriesProperties.attributeName,
+                ch_name=alpha_path_utils.parse_path(attr.stringSeriesProperties.attributeName)[-1],
```

Comment:
Shouldn't be `[:-1]` insted of `[-1]`?

Reply:
Shouldn't `ch_name` be string instead of list?

Reply:
But it is string. `[-1]` returns last element from list, so example values used in our code could be:
```
ch_id='monitoring/stdout',
ch_name='stdout',
```

Reply:
We do, right.